src/Avalonia.Base/Input/GestureRecognizers/VelocityTracker.cs

```diff
@@ -0,0 +1,402 @@
+﻿// Code in this file is derived from
+// https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/gestures/velocity_tracker.dart
+
+//Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Instead of license header in this file, it should be added to our [NOTICE.md](https://github.com/AvaloniaUI/Avalonia/blob/master/NOTICE.md).

"Code in this file is derived from" part should be enough to keep here.